liota/core/package_manager.py

```diff
@@ -518,14 +515,23 @@ def _package_load(self, file_name, ext_forced=None, check_stack=None):
             log.warning("Package already loaded: %s" % file_name)
             return None
 
-        path_file_ext, file_ext = self._package_chk_exists(
+        path_file_ext, file_ext, checksum_list = self._package_chk_exists(
```

Comment:
My first thought was also like what you said. But we have not only those cmds, but also support pkg_list as its cmd input, and of course we also have packages_auto.txt which will have package list and will be loaded when package manager is started. So I had thought users might feel troublesome to have to calculate hash first and then fill them into that file. 

Anyway, I had tried to avoid above troublesome for both users and package manager.
But if you think that's the only way, I can then modify in that way.